### ISSUE TITLE
Poor  UI After Log In?

### ISSUE THREAD

**Issue by shirshak55** [httptoolkit/httptoolkit#366]
<img width="1369" alt="image" src="https://user-images.githubusercontent.com/8097377/206835624-3abc52e3-221b-44f3-b68b-a1c52d9adf84.png">

I can't easily scroll and I don't see Get Pro button.

**Comment by pimterry** [comment]
Yes, that's definitely broken. I'm sure this happened in the past on smaller screens and I tweaked it to fit... Something's clearly changed and broken this anyway. I've just pushed a fix, so it should be working properly from now on. Thanks for reporting this!

(In your case though of course, as a contributor you shouldn't be seeing this screen at all! I think your Pro account is linked to your @pm.me address though. Let me know if you'd like that switched to your gmail instead).
